nautobot/extras/templates/extras/configcontext_edit.html

```diff
@@ -28,7 +28,7 @@
             {% render_field form.device_redundancy_groups %}
             {% render_field form.tags %}
             {% if settings.CONFIG_CONTEXT_DYNAMIC_GROUPS_ENABLED %}
-            {% render_field form.dynamic_groups %}
+            {% render_field form.dynamic_groups_cached %}
```

Comment:
I think `dynamic_groups` here is ConfigContext.dynamic_groups. Possibly not good test coverage here.
```suggestion
            {% render_field form.dynamic_groups %}
```